chain-extensions/xvm/src/lib.rs

```diff
@@ -18,19 +18,15 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-use astar_primitives::xvm::{CallError, Context, VmId, XvmCall};
+use astar_primitives::xvm::{Context, VmId, XvmCall};
 use frame_support::dispatch::Encode;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `astar_primitives::xvm::{Context, VmId, XvmCall}` is missing. Please add it.
```diff
+use astar_primitives::xvm::{Context, VmId, XvmCall};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->